modules/consensus/synchronize_test.go

```diff
@@ -438,12 +438,12 @@ func TestIntegrationRPCSendBlocks(t *testing.T) {
 		// Create the "remote" peer.
 		remoteCST, err := blankConsensusSetTester(filepath.Join(t.Name()+" - remote", strconv.Itoa(i)))
 		if err != nil {
-			t.Fatal(err)
+			t.Fatalf("%v: %s", tt.msg, err)
```

Comment:
interesting. There is another reason to use `%v` though, which is that `%s` produces garbage output if `err == nil`:
```go
fmt.Printf("%s", error(nil))
// output:
// %!s(<nil>)
```